Makefile

```diff
@@ -5,7 +5,12 @@ all:
 
 .PHONY: gen-proto
 gen-proto:
+	rm -rf idl/{google,grafeas,validate} || true
```

Comment:
Wasn't sure how to exclude these dependencies from the linter, so I settled with nuking them and then restoring them with `buf export` below